### DISCUSSION TITLE
When deploying on minikube ingress address is blank

### DISCUSSION THREAD

**Discussion by craftzneko** [Budibase/budibase#6620] [Help]
running through this guide to try budibase on minikube https://docs.budibase.com/docs/kubernetes-k8s

All works up to when i run kubectl get ingress --namespace budibase

it returns a blank address, new to kube so what am i missing here?

**Comment by craftzneko** [comment]
tried port forward and that works and i can create account but errors loading apps

**Reply by melohagan** [reply]
Hi @pr1malc0de Great that the port forwarding worked! What errors are you getting when loading apps? (You can press Fn+F12 on Chrome to see the errors in the console & network tabs).

Are your issues still persisting?
